azure/nightly-int-qa-pipeline.yml

```diff
@@ -26,9 +26,15 @@ variables:
   - name: SERVICE_ARTIFACT_NAME
     value: ''
 
-extends:
-  template: ./templates/run-tests.yml
-  parameters:
-    nightly: true
-    environment: 'internal-qa'
-    test_command: 'make internal-qa-test'
+steps:
+  - template: ./templates/run-tests.yml
+    parameters:
+      nightly: true
+      environment: 'internal-qa'
+      test_command: 'make internal-qa-test'
+  - bash: |
+      curl -X POST $(ALERTS_PROD_API_WEBHOOK_URI) \
+      -H 'Content-Type: application/json' \
+      -d '{"@type":"MessageCard","@context":"http://schema.org/extensions","themeColor":"ff0000","title":"Nightly Internal QA Test Run Failed","text":"https://dev.azure.com/NHSD-APIM/API%20Platform/_build/results?buildId='$(build.buildid)'&view=results"}'
```

Comment:
I wonder if you want to template this alerting so we don't replicate the action for sending the message?